bands/templates/band_admin/band_info.html

```diff
@@ -3,262 +3,115 @@
 {% block content %}
 
 <style type="text/css">
-    .col-sm-6 {
-        font-family: Courier;
-        padding-top: 8px;
-    }
+    .form-group { margin-bottom: 8px; }
 </style>
 
 <h2>Band Info for <a href="../bands/index?id={{ band.id }}">{{ band.group.name }}</a></h2>
 
-<br/> <br/>
 <form class="form-horizontal" role="form" method="post" action="band_info">
     {{ csrf_token() }}
     <input type="hidden" name="id" value="{{ band.id }}" />
 
-    {% if band.panel.name %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Name:</label>
-        <div class="col-sm-6">
-            {{ band.panel.name }}
-        </div>
-    </div>
-    {% endif %}
-
-    {% if band.panel.length %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Length:</label>
-        <div class="col-sm-6">
-            {{ band.panel.length }}
-        </div>
-    </div>
-    {% endif %}
-
-    {% if band.panel.desc %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Description:</label>
-        <div class="col-sm-6">
-            {{ band.panel.desc|linebreaksbr }}
-        </div>
-    </div>
-    {% endif %}
-
-    {% if band.panel.tech_needs %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Tech Needs:</label>
-        <div class="col-sm-6">
-            {{ band.panel.panel_tech_needs_labels|join(" / ") }}
-        </div>
-    </div>
-    {% endif %}
+    {{ macros.form_group(band.panel, 'name', label='Panel Name', is_readonly=True) }}
+    {{ macros.form_group(band.panel, 'length', label='Panel Length', is_readonly=True) }}
+    {{ macros.form_group(band.panel, 'desc', label='Panel Description', is_readonly=True) }}
 
-    {% if band.bio.desc %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Bio:</label>
-        <div class="col-sm-6">
-            {{ band.bio.desc|linebreaksbr }}
-        </div>
-    </div>
+    {% if band.panel.tech_needs or band.panel.other_tech_needs %}
+      {{ macros.form_checkgroup(band.panel, 'tech_needs',
+          other_field='other_tech_needs',
+          label='Panel Tech Needs',
+          is_required=True,
```

Comment:
Although, after a quick test, I can see that `is_required` is not actually needed, so I'll remove it.